bin/adhoc-scripts/ParseSpecCmsswdist.py

```diff
@@ -227,7 +244,12 @@ def getPipVersion(specdir, specfile):
         line0 = lines[0].strip()
         nameAndVer = []
         if "### RPM external" in line0:
-            nameAndVer = line0[line0.find("-") + 1:].split(" ")
+            # nameAndVer = line0[line0.find("-") + 1:].split(" ")
+            line0 = line0.split(" ")
```

Comment:
here the `line0` changed from string type to array type. This is very bad practice. Please use a different name, e.g.
```
arr = lin0.split(" ")
```

Reply:
fixed